packages/nx/src/command-line/release/plan.ts

```diff
@@ -355,7 +356,7 @@ async function _promptForMessage(versionPlanName: string): Promise<string> {
       );
       writeFileSync(messageFilePath, '');
       await launchEditor(messageFilePath);
-      message = readFileSync(messageFilePath, 'utf-8');
+      message = readFileSync(messageFilePath, 'utf8');
```

Comment:
`utf-8` is already a valid encoding value and it's more readable, please revert

```
type BufferEncoding =
            | "ascii"
            | "utf8"
            | "utf-8"
            | "utf16le"
            | "ucs2"
            | "ucs-2"
            | "base64"
            | "base64url"
            | "latin1"
            | "binary"
            | "hex";
```

Reply:
I've changed this to match the Node docs, but I'm fine with reverting. 😄